db/migrate/20190412003025_create_users.rb

```diff
@@ -0,0 +1,17 @@
+class CreateUsers < ActiveRecord::Migration[5.2]
+  def change
+    create_table :users do |t|
+      t.string :name
```

Comment:
It might be easier to have a first name and a last name

Reply:
I could not think of a good reason to have two different variables for first name and last name.

Reply:
changed as requested